package.json

```diff
@@ -58,7 +58,7 @@
     "ngx-bootstrap": "^3.0.1",
     "ngx-clipboard": "^12.2.1",
     "ngx-markdown": "^8.1.0",
-    "pipeline-builder": "^0.3.10-dev.298",
+    "pipeline-builder": "^0.3.10-dev.313",
```

Comment:
Do they have non-dev releases?
(Is this the equivalent on a SNAPSHOT in Maven?)

Reply:
They do, but it hasn't been updated in forever

Reply:
:disappointed: